File: 05 SWEA/D1/Java/src/Prob2019.java

/**
* title  : 2019. 더블더블 (D1)
* time   : 96ms
* memory : 16884kb
*/

import java.io.*;
 
public class Prob2019 {
    
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();

        int N = Integer.parseInt(br.readLine());
        int answer = 1;

        sb.append(answer).append(" ");
        for (int i = 1; i <= N; i++) {
            answer *= 2;
            sb.append(answer).append(" ");
        }

        System.out.println(sb);
    }
}

File: 05 SWEA/D1/Python/Prob2019.py

"""
title  : 2019. 더블더블 (D1)
time   : 114ms
memory : 43236kb
"""

def solution():
    
    N = int(input())

    answer = 1
    print(answer, end=" ")
    
    for _ in range(N):
        answer *= 2
        print(answer, end=" ")

solution()